public/museu/evolution-of-tabnews.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>A Evolução do TabNews</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300;400;500;700&display=swap" rel="stylesheet"> 
+    <style>
+        html{
+            background-color: #f5f5f5;
+            max-width: 100vw;
+            font-family: 'Ubuntu', sans-serif;
+            font-size: 1rem;
+        }
+
+        main{
+            margin: 0 auto;
+            width: 80vw;
+            padding-top: 50px;
+            padding-bottom: 50px;
+        }
+
+        img{
+            max-width: 100%;
+        }
+
+        .title{
+            text-align: center;
+        }
+
+        .green{
+            color: green;
+        }
+
+        a{
+            color: rgb(0, 85, 255);
+        }
+
+        a:hover{
+            color: rgba(0, 85, 255, 0.639);
+            transition: 0.3s;
+        }
+
+        .footer-text{
+            color: rgba(0, 0, 0, 0.815);
+            text-align: end;
+        }
+    </style>
+</head>
+<body>
+    <main>
+        <h1 class="title">A Evolução do TabNews</h1>
+        <p>Algumas (ou muitas) pessoas estão chegando depois do vídeo de lançamento do TabNews no Youtube. Além disso, muitas pessoas chegaram já com essa interface atual, então eu resolvi criar uma &quot;timeline&quot; com as principais mudanças desde o início, não só do site, mas das idéias.</p>
```

Comment:
Como isso entrará para o museu, acredito que seja legal adicionar um link para o vídeo de lançamento: 

```suggestion
        <p>Algumas (ou muitas) pessoas estão chegando depois do <a href="https://www.youtube.com/watch?v=bYBVCxVwrdY">vídeo de lançamento do TabNews no Youtube</a>. Além disso, muitas pessoas chegaram já com essa interface atual, então eu resolvi criar uma &quot;timeline&quot; com as principais mudanças desde o início, não só do site, mas das ideias.</p>
```

PS: `ideais` não possui mais acento agudo.